utils/mil_tools/nodes/online_bagger.py

```diff
@@ -47,7 +47,16 @@ def __init__(self):
                                              self.start_bagging)
 
         self.subscribe_loop()
-        rospy.loginfo('subscriber list: %s', self.subscriber_list)
+
+        rospy.loginfo('')
+        rospy.loginfo('Remaining Failed Topics:')
+        self.print_subscriber_list(False)
+        rospy.loginfo('')
+
+    def print_subscriber_list(self, status):
```

Comment:
Add a docstring, and if you want to use my suggestion above, make this function not print anything, just return a string.